packages/service-library/src/servicelib/long_running_tasks/_task.py

```diff
@@ -187,9 +187,21 @@ def _get_tracked_task(
                 if with_task_context and (
                     tasks[task_id].task_context != with_task_context
                 ):
+                    logger.debug(
```

Comment:
Q: I see that both these raise an error. Why don't you see the traceback?

Reply:
I was wondering that myself. Could it be because I don't search the correct way in Graylog? When I look for the logs I search 
"<task_id>"
in Graylog. That search gives me a lot of log messages in Graylog related to that task. But I don't see the tracebacks of these exceptions being raised.

Reply:
I'm not familiar with the code but these what I think it could be:
- if this is a task, you need to retrieve it's result and do something with them (log them)
- maybe this is not a task and the http call serialises the error directly (in this situation it is OK to log the result like this)